mathjax3-ts/output/chtml/FontData.ts

```diff
@@ -0,0 +1,328 @@
+export const H = 'H';
+
+/****************************************************************************/
+/*
+ *  The FontData class (for storing character bounding box data by variant,
+ *                      and the stretchy delimiter data).
+ */
+export class FontData {
+
+    /*
+     *  The standard variants to define
+     */
+    protected static defaultVariants = [
+        ['normal'],
+        ['bold', 'normal'],
+        ['italic', 'normal'],
+        ['bold-italic', 'italic', 'bold'],
+        ['double-struck', 'bold'],
+        ['fraktur', 'normal'],
+        ['bold-fraktur', 'bold', 'fraktur'],
+        ['script', 'normal'],
+        ['bold-script', 'bold', 'script'],
+        ['sans-serif', 'normal'],
+        ['bold-sans-serif', 'bold', 'sans-serif'],
+        ['sans-serif-italic', 'italic', 'sans-serif'],
+        ['bold-sans-serif-italic', 'bold-italic', 'sans-serif'],
+        ['monospace', 'normal']
+    ];
+
+    /*
+     *  The default font parameters for the font
+     */
+    public static defaultParams: FontParameters = {
+        x_height:         .442,
+        quad:             1,
+        num1:             .676,
+        num2:             .394,
+        num3:             .444,
+        denom1:           .686,
+        denom2:           .345,
+        sup1:             .413,
+        sup2:             .363,
+        sup3:             .289,
+        sub1:             .15,
+        sub2:             .247,
+        sup_drop:         .386,
+        sub_drop:         .05,
+        delim1:          2.39,
+        delim2:          1.0,
+        axis_height:      .25,
+        rule_thickness:   .06,
+        big_op_spacing1:  .111,
+        big_op_spacing2:  .167,
+        big_op_spacing3:  .2,
+        big_op_spacing4:  .45, // .6,  // better spacing for under arrows and braces
+        big_op_spacing5:  .1,
+
+        surd_height:      .075,
+
+        scriptspace:         .05,
+        nulldelimiterspace:  .12,
+        delimiterfactor:     901,
+        delimitershortfall:   .3,
+
+        min_rule_thickness:  1.25     // in pixels
+    };
+
+    /*
+     * The default delimiter and character data
+     */
+    protected static defaultDelimiters: DelimiterMap = {};
+    protected static defaultChars: CharMapMap = {};
+
+    /*
+     * The default variants for the fixed size stretchy delimiters
+     */
+    protected static defaultSizeVariants: string[] = [];
+
+    /*
+     * The actual variant, delimiter, and size information for this font
+     */
+    protected variant: VariantMap = {};
+    protected delimiters: DelimiterMap = {};
+    protected sizeVariants: string[];
+
+    /*
+     * The actual font parameters for this font
+     */
+    public params: FontParameters;
+
+    /*
+     * Copies the data from the defaults to the instance
+     *
+     * @constructor
+     */
+    constructor() {
+        let CLASS = (this.constructor as typeof FontData);
+        this.params = Object.assign({}, CLASS.defaultParams);
```

Comment:
Use spread: 
this.params = {...Class.defaultParams};
(At least I believe that was the syntax.

Reply:
Yes, that's right.  I'll change it.